pkg/roachprod/vm/vm.go

```diff
@@ -508,8 +508,10 @@ type Provider interface {
 	// GetPreemptedSpotVMs returns a list of Spot VMs that were preempted since the time specified.
 	// Returns nil, nil when SupportsSpotVMs() is false.
 	GetPreemptedSpotVMs(l *logger.Logger, vms List, since time.Time) ([]PreemptedVM, error)
-	// GetHostErrorVMs returns a list of Spot VMs that had host error since the time specified.
+	// GetHostErrorVMs returns a list of VMs that had host error since the time specified.
 	GetHostErrorVMs(l *logger.Logger, vms List, since time.Time) ([]string, error)
+	// GetVMSpecs returns a json list of VM specs
```

Comment:
Nit: "...json list of VM specs, according to a specific cloud provider. E.g., GCE [1], AWS [2].

[1] https://cloud.google.com/sdk/gcloud/reference/compute/instances/describe
[2] https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html"